src/pages-and-resources/discussions/app-list/AppListNextButton.jsx

```diff
@@ -17,7 +17,12 @@ function AppListNextButton({ intl }) {
   }, [discussionsPath, selectedAppId]);
 
   return (
-    <Button variant="primary" onClick={handleStartConfig} className="mr-2">
+    <Button
+      aria-label={intl.formatMessage(messages.nextButton)}
```

Comment:
It's my understanding that paragon provided accessible components. so why we need to provide `aria-label` here? I believe this is used for a11y - and the paragon button should apply that out of the box? no?

Reply:
Is this `aria-label` can be different? I ask b/c in this case the same text is being used for aria and button text.

Reply:
Paragon doesn't automatically add an `aria-label` to Buttons because often the button text itself acts as a sufficient label. 

That said, I'm using it wrong here and will change it - because it's the same as the button text, I don't need these labels.  I'm going to remove them and then use the button text to find the buttons in my tests.  

Good question!